sdk/core/System.ServiceModel.Rest/src/Pipeline/HttpPipelineMessageTransport.cs

```diff
@@ -59,21 +59,33 @@ private static HttpClient CreateDefaultClient()
         };
     }
 
-    public override PipelineMessage CreateMessage()
+    public override PipelineMessage CreateMessage(InvocationOptions options)
     {
         PipelineRequest request = new HttpPipelineRequest();
         PipelineMessage message = new PipelineMessage(request);
 
+        // Wire up options on message
+        message.CancellationToken = options.CancellationToken;
+        message.ResponseClassifier = options.ResponseClassifier;
```

Comment:
Note: I'm not sure there is anything "transport specific" about wiring the options here.  Maybe Transport doesn't need to take `TMessageOptions` and the pipeline can do any options-wiring needed?